src/screens/Sidebar/Sidebar.res

```diff
@@ -407,13 +410,19 @@ module SidebarNestedSection = {
     <RenderIf condition={!areAllSubLevelsHidden}>
       <NestedSectionItem
         section
-        isSectionExpanded
+        isSectionExpanded={isSectionAutoCollapseEnabled
```

Comment:
Put this condition out for better readabiluty and use If else for the same

Reply:
done